modules/ROOT/pages/dw-core-annotations.adoc

```diff
@@ -1,61 +1,98 @@
 = Core Annotations (dw::Core)
 
-
-[%header,cols="2,2,4a"]
+[%header, cols="1,1,1a"]
 |===
 | Annotation | Definition | Description
 
 | AnnotationTarget
-| `@AnnotationTarget(targets: Array<String>)`
-| Defines where the Annotation can be used. If not defined, it can be used in any possible target.
-
-
-Options are:
+| `@AnnotationTarget&#40;targets: Array<"Function" &#124; "Parameter" &#124; "Variable" &#124; "Import"&#62;&#41;`
```

Comment:
TODO: copyedit descriptions of annotations in DW source, including some extraneous "Introduced in DataWeave version 2.4.0. Supported by Mule ..." in the source that are different from the other "Introduced in" blurbs.